docs/reference/eql/eql-search-api.asciidoc

```diff
@@ -0,0 +1,173 @@
+(EQL)] queries on logs and other event-based data in {es}.
+
+You can use EQL to:
+
+* Find a sequence of events that meet specific conditions, with an optional time
+  constraint.
+* Match events with shared properties
+* Quickly filter and aggregate event data using a pipe syntax
+
+EQL is often used for security use cases, such as threat detection and log
+monitoring.
+
+[[eql-search-api-desc-limitations]]
+===== Limitations
+
+Lineage keywords::
+The EQL search API cannot run EQL queries containing
+https://eql.readthedocs.io/en/latest/query-guide/basic-syntax.html#event-relationships[lineage-related keywords]:
+
+* `child of`
+* `descendant of`
+* `event of`
+
+Array functions::
+The EQL search API cannot run EQL array functions:
+
+* https://eql.readthedocs.io/en/latest/query-guide/functions.html#arrayContains[`arrayContains`]
+* https://eql.readthedocs.io/en/latest/query-guide/functions.html#arrayCount[`arrayCount`]
+* https://eql.readthedocs.io/en/latest/query-guide/functions.html#arraySearch[`arraySearch`]
+
+[[eql-search-api-path-params]]
+==== {api-path-parms-title}
+
+`<index>`::
++
+--
+(Optional, string)
+Comma-separated list of index names or <<indices-aliases,index aliases>> used to
+limit the request. Accepts wildcard expressions.
+
+To search all indices, use `_all`.
+--
+
+[[eql-search-api-query-params]]
+==== {api-query-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
++
+Defaults to `false`.
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=expand-wildcards]
++
+Defaults to `open`.
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailable]
+
+[[eql-search-api-request-body]]
+==== {api-request-body-title}
+
+`rule`::
+(Required, string)
+https://eql.readthedocs.io/en/latest/query-guide/index.html[EQL] query you wish
+to run.
+
+`event_type_field`::
++
+--
+(Required*, string)
+Field containing the event classification, such as `process`, `file`, or
+`network`.
+
+Defaults to `event.category`, as defined in the {ecs-ref}/ecs-event.html[ECS].
+If an index does not contain the `event.category` field, this value is
+required.
+--
+
+`timestamp_field`::
++
+--
+(Required*, string)
+Field containing event timestamp.
+
+Defaults to `@timestamp`, as defined in the {ecs-ref}/ecs-base.html[ECS]. If an
+index does not contain the `@timestamp` field, this value is required.
+--
+
+`implicit_join_key_field`::
++
+--
+(Required*, string)
+Default field used for
+https://eql.readthedocs.io/en/latest/query-guide/joins.html[EQL joins].
+
+You can override this value by explicitly specifying one or more join key fields
+in the EQL query.
+
+Defaults to `agent.id`, as defined in the {ecs-ref}/ecs-agent.html[ECS]. If an
+index does not contain the `agent.id` field, this value is required.
+--
+
+`query`::
+(Optional, <<query-dsl,query DSL object>>)
+Query, written in query DSL, used to filter the documents on which the EQL query
+runs.
```

Comment:
I plan to add an example for this parameter once the response format is settled. Using a `range` query to narrow dates might work well.